fabric-input-events-v1/src/main/java/net/fabricmc/fabric/impl/client/input/FabricKeyboardImpl.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.client.input;
+
+import net.minecraft.client.util.InputUtil;
+import net.minecraft.client.util.InputUtil.Key;
+
+public class FabricKeyboardImpl {
+	public static FabricKeyboardImpl INSTANCE = new FabricKeyboardImpl();
+
+	private int modKeys = 0;
+
+	private FabricKeyboardImpl() {
+	}
+
+	public boolean isKeyPressed(int keycode, int scancode) {
+		return InputUtil.isKeyPressed(keycode, scancode);
```

Comment:
Why having methods for that if `InputUtil` already provides them?

Reply:
Just for putting everything in one place, `isKeyPressed(Key)` is generally more useful. `FabricKeyboard` and `FabricMouse` are for the bits of information that Minecraft doesn't provide.